content/docs/user-guide/index.md

```diff
@@ -56,13 +87,30 @@ bringing best practices from software engineering into the data science field.
 
 ### Git
 
-- DVC builds upon Git by introducing the concept of data files – large files
-  that should not be stored in a Git repository, but still need to be tracked
-  and versioned. It leverages Git's features to enable managing different
-  versions of data, data pipelines, and experiments.
+DVC builds upon Git by introducing the concept of _data versioning_ – large
+files that should not be stored in a Git repository, but still need to be
+tracked and versioned. It leverages Git's features to enable managing different
+versions of data, data pipelines, and experiments.
+
+<admon type="info">
+
+DVC is not fundamentally bound to Git, and can work without it (except
+[versioning-related](/doc/use-cases/versioning-data-and-models) features).
```

Comment:
Experiments also don't work without Git 😬

Reply:
Yeah we have this note in a couple places and I thought about updating it but not sure exactly which features will and won't. Is there a more general way to state it? Like "versioning-related features won't work" ?